### ISSUE TITLE
[duktape] AppSceneContext.js not in sync with latest master

### ISSUE THREAD

**Issue by madanagopaltcomcast** [pxscene/pxCore#798]
**Steps to reproduce:**
./pxscene.sh http.js

**Output seen:**
[LOG INFO] AppSceneContext#loadScenePackage1 done
[LOG ERROR] Error - failed to get Remote modules for: https.js, error=ReferenceError: identifier 'isLocalApp' undefined
[LOG ERROR] Main module[http.js] load has failed - on failed imports: , err=ReferenceError: identifier 'isLocalApp' undefined

**Expected output:**
No error seen

**Initial analysis:**
Basically, isLocalApp() is defined in AppSceneContext.js in latest master. We need to keep the code in sync with latest master.
